parlai/scripts/convert_fairseq_to_parlai.py

```diff
@@ -0,0 +1,485 @@
+#!/usr/bin/env python3
```

Comment:
hmm, should it live parlai scripts? I really wonder if this shouldn't be tucked a bit deeper.

Reply:
happy to put it in `parlai/agents/bart/` if you prefer. or perhaps a utils file?